### ISSUE TITLE
PHP fatal error when clicking new user activation link for multisites

### ISSUE THREAD

**Issue by adamdunnage** [google/site-kit-wp#7085]
A [user in the support forum](https://wordpress.org/support/topic/cannot-activate-new-user-accounts-with-activate-link/#post-16768202) is reporting PHP fatal errors when users invited to a multisite are clicking the new user activation link sent via email. This was reported with PHP 8 and WordPress 6.2.2. Downgrading to PHP 7.4 stops the PHP fatal errors in this scenario. 

Steps the user followed:

1. From multisite network admin go to **Users > Add new**.
2. Fill out everything under the **Add New User** section making sure to leave the 'Skip Confirmation Email' checkbox unselected. Note: make sure you have access to the email the confirmation will go to for testing. 
3. Add the new user.
4. Navigate to the activation email and click on the activation link. Note: the link will look similar to this - http: //www. Domain. Com/wp-activate. Php? Key=879df7gd9f

This has not been reproducible when using our own support multisite setup with PHP 8 and Wordpress 6.2.2 and following the steps above. 

The error reported in this case is as follows:

```
Error Log when trying to activate a user:
PHP Fatal error: Uncaught TypeError: base64_decode(): Argument #1 ($string) must be of type string, array given in /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Storage/Data_Encryption.php:90\nStack trace:\n#0 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Storage/Data_Encryption.php(90): base64_decode(Array, true)\n#1 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Storage/Encrypted_Options.php(78): Google\\Site_Kit\\Core\\Storage\\Data_Encryption->decrypt(Array)\n#2 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Authentication/Credentials.php(65): Google\\Site_Kit\\Core\\Storage\\Encrypted_Options->get(‘googlesitekit_c…’)\n#3 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Authentication/Credentials.php(77): Google\\Site_Kit\\Core\\Authentication\\Credentials->get()\n#4 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Authentication/Authentication.php(1384): Google\\Site_Kit\\Core\\Authentication\\Credentials->has()\n#5 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Util/Method_Proxy_Trait.php(25): Google\\Site_Kit\\Core\\Authentication\\Authentication->filter_features_via_proxy(false, ‘dashboardSharin…’)\n#6 /nas/content/live/nahl/wp-includes/class-wp-hook.php(308): Google\\Site_Kit\\Core\\Authentication\\Authentication->Google\\Site_Kit\\Core\\Util\\{closure}(false, ‘dashboardSharin…’)\n#7 /nas/content/live/nahl/wp-includes/plugin.php(205): WP_Hook->apply_filters(false, Array)\n#8 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Util/Feature_Flags.php(56): apply_filters(‘googlesitekit_i…’, false, ‘dashboardSharin…’)\n#9 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Modules/Modules.php(221): Google\\Site_Kit\\Core\\Util\\Feature_Flags::enabled(‘dashboardSharin…’)\n#10 /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Plugin.php(161): Google\\Site_Kit\\Core\\Modules\\Modules->register()\n#11 /nas/content/live/nahl/wp-includes/class-wp-hook.php(308): Google\\Site_Kit\\Plugin->Google\\Site_Kit\\{closure}(”)\n#12 /nas/content/live/nahl/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(”, Array)\n#13 /nas/content/live/nahl/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#14 /nas/content/live/nahl/wp-settings.php(623): do_action(‘init’)\n#15 /nas/content/live/nahl/wp-config.php(135): require_once(‘/nas/content/li…’)\n#16 /nas/content/live/nahl/wp-load.php(50): require_once(‘/nas/content/li…’)\n#17 /nas/content/live/nahl/wp-activate.php(12): require(‘/nas/content/li…’)\n#18 {main}\n thrown in /nas/content/live/nahl/wp-content/plugins/google-site-kit/includes/Core/Storage/Data_Encryption.php on line 90
```

**Support topic:** https://wordpress.org/support/topic/cannot-activate-new-user-accounts-with-activate-link/#post-16768202

**Shared Site Health Information:** https://docs.google.com/forms/d/1lhjns2K7c0Ny7ME4gxTcsnYtNUtcMr8BvFpPQi23Y1E/edit#response=ACYDBNhb7dVvZU1jCqBEJpwiRMWJb3-MZsURqFyPwiRLpHSbp24oHXGn7aDj5KSTjYsayeY

**Comment by josip-matic** [comment]
Hi, I have the same problem on my multisite. This is some kind of bug inside wp. Link example:
_https://domain/something/something/wp-activate.php?key=123_key_678_ but if you try
_https://domain/wp-activate.php?key=123_key_678_ then work fine

**Comment by adamdunnage** [comment]
@josip-matic Thanks for getting in touch. We don't provide support through GitHub so could I please ask you to [open a new support topic](https://wordpress.org/support/plugin/google-site-kit/#new-topic-0) with us and we will be happy to troubleshoot this with you, thanks!

**Comment by Omnitech-cse** [comment]
I reproduce this exact same issue on my development multisite. It is very annoying.

**Comment by jamesozzie** [comment]
@adamdunnage I think this can be closed as a duplicate of https://github.com/google/site-kit-wp/issues/7653 

Feel free to review. There is also a support topic associated with this GitHub issue which you may wish to update.

**Comment by adamdunnage** [comment]
> @adamdunnage I think this can be closed as a duplicate of #7653
> 
> Feel free to review. There is also a support topic associated with this GitHub issue which you may wish to update.

As stated by @jamesozzie, this is a duplicate of [#7653](https://github.com/google/site-kit-wp/issues/7653). Closing this issue.
